apps/native-component-list/src/navigation/MainNavigators.tsx

```diff
@@ -29,4 +33,13 @@ export default {
     },
     navigator: ExpoComponentsStackNavigator,
   },
-};
+} as Record<
+  string,
+  | undefined
```

Comment:
Why this `undefined` here? 🤔

Reply:
Because not for every `string` there will be a value? I wanted to make it clear this way.

Reply:
FWIW [`Record` docs](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeystype) define `Keys` as an explicit union type.